src/services/TestResultsService.ts

```diff
@@ -364,38 +369,40 @@ export class TestResultsService {
 
   public insertTestResultsList(testResultsItems: ITestResult[]) {
     return this.testResultsDAO.createMultiple(testResultsItems)
-      .then((data: any) => {
-        if (data.UnprocessedItems) { return data.UnprocessedItems; }
-      })
-      .catch((error: any) => {
-        if (error) {
-          console.log("Error in insertTestResultsList: ", error);
-          throw new HTTPError(500, MESSAGES.INTERNAL_SERVER_ERROR);
-        }
-      });
+        .then((data: any) => {
+          if (data.UnprocessedItems) {
+            return data.UnprocessedItems;
+          }
+        })
+        .catch((error: any) => {
+          if (error) {
+            console.log("Error in insertTestResultsList: ", error);
+            throw new HTTPError(500, MESSAGES.INTERNAL_SERVER_ERROR);
+          }
+        });
   }
 
   public deleteTestResultsList(testResultsVinIdPairs: any[]) {
     return this.testResultsDAO.deleteMultiple(testResultsVinIdPairs)
-      .then((data: any) => {
-        if (data.UnprocessedItems) {
-          return data.UnprocessedItems;
-        }
-      })
-      .catch((error: any) => {
-        if (error) {
-          console.log(error);
-          throw new HTTPError(500, MESSAGES.INTERNAL_SERVER_ERROR);
-        }
-      });
+        .then((data: any) => {
+          if (data.UnprocessedItems) {
+            return data.UnprocessedItems;
+          }
+        })
+        .catch((error: any) => {
+          if (error) {
+            console.log(error);
+            throw new HTTPError(500, MESSAGES.INTERNAL_SERVER_ERROR);
+          }
+        });
   }
 
-  public lecTestTypeWithoutCertificateNumber(payload: { testStatus?: string; testTypes: any; vehicleType?: any; vehicleSize?: any; vehicleConfiguration?: any; noOfAxles?: any; }) {
-    const bool = false;
+  public shouldContainCertificateNumber (payload: ITestResultPayload): boolean {
```

Comment:
Function name doesn't really describe what it does - "isMissingRequiredCertificateNumber" would be better.

Reply:
fixed